src/System.Drawing.Common/tests/Helpers.cs

```diff
@@ -47,8 +47,8 @@ public static bool GetGdiplusIsAvailable()
 
         public static bool GetRecentGdiPlusIsAvailable2()
         {
-            // CentOS 7, RHEL 7 and Ubuntu 14.04, as well as Fedora 25 and OpenSUSE 4.22 are running outdated versions of libgdiplus
-            if (PlatformDetection.IsCentos7 || PlatformDetection.IsRedHat || PlatformDetection.IsUbuntu1404 || PlatformDetection.IsFedora || PlatformDetection.IsOpenSUSE)
+            // RedHat and Ubuntu 14.04, as well as Fedora 25 and OpenSUSE 4.22 are running outdated versions of libgdiplus
+            if (PlatformDetection.IsRedHatFamily || PlatformDetection.IsUbuntu1404 || PlatformDetection.IsFedora || PlatformDetection.IsOpenSUSE)
```

Comment:
>PlatformDetection.IsRedHatFamily [](start = 16, length = 32)

we should check version 7 here.

Reply:
interesting, I am seeing the original code was checking for redhat without the version. If you are not getting any problem with your change here, then I think your change would be ok.